src/h5web/metadata-viewer/LinkInfo.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { HDF5Link, HDF5LinkClass } from '../providers/models';
+
+interface Props {
+  link: HDF5Link;
+}
+
+function LinkInfo(props: Props): JSX.Element {
+  const { link } = props;
+
+  return (
+    <>
+      <tr>
+        <th colSpan={2}>Link info</th>
```

Comment:
I would add the styles.table_head here for consistency with the other table heads.